app.js

```diff
@@ -17,7 +13,7 @@ var config = {
 };
 
 
-require(path.join(__dirname, "index.js")).connect(function (context) {
+require(path.join('./index.js')).connect(function (context) {
```

Comment:
@Sandr00 I can't reproduce your issue. Both `npm start` and `npm test` run without an issue for me.